tests/pytest_tests/system_tests/test_artifacts/test_wandb_artifacts_api.py

```diff
@@ -36,6 +36,31 @@ def test_fetching_artifact_files(user, wandb_init):
     assert open(file_path).read() == "testing"
 
 
+def test_artifact_download_offline_mode(user, wandb_init):
+    project = "test"
+
+    with wandb_init(entity=user, project=project) as run:
+        artifact = wandb.Artifact("test-artifact", "test-type")
+        with open("boom.txt", "w") as f:
+            f.write("testing")
+        artifact.add_file("boom.txt", "test-name")
```

Comment:
you are going to create a file in the location where this test is executed, and it won't be cleaned, meaning you leak state to other tests. The only reason it work here, is because we have `filesystem_isolate` fixture that scoped to a function and is auto used

Reply:
I did see many other examples following this pattern, but I get your point. Will change this to use mock and temp directory 👍